### PR TITLE
Feat/shopping list items controller

### PR DESCRIPTION
**Pull request by isadorabk**

### CODE REVIEW

File: controllers/ingredients.controller.js [hunk 1]
```diff
@@ -35,9 +35,18 @@ class IngredientsController {
       } else {
         // If there's no ingredient, create a new one
         ingredient = filterProps(data, ['name', 'ingredient_type_id']);
-        let newIngredient = await this.Ingredient.create(ingredient);
+        const newIngredient = await this.Ingredient.create(ingredient);
+        let res = filterProps(newIngredient.dataValues, ['id', 'name']);
```
Comment (arol) [inline controllers/ingredients.controller.js]: `const`

10 push-ups

File: controllers/plans.controller.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+  async updateUsersPlanById (ctx, next) {
+    // Check if the method is correct
+    if (ctx.method !== 'PUT') throw new Error('Method not allowed');
+
+    const data = ctx.request.body;
+    data.name = data.name.toLowerCase();
+    
+    // Update the plan with the specific id
+    let plan = filterProps(data, ['name']);
```
Comment (arol) [inline controllers/plans.controller.js]: `const`

File: controllers/users.controller.js [hunk 3]
```diff
@@ -46,11 +47,29 @@ class UsersController {
...
         const token = jwt.sign({
-          id: res.id
+          id: res.id,
         }, process.env.JWT_SECRET, {
           expiresIn: 2592000
         });
         res.token = token;
+
+        // Create first empty plan for the user
+        let plan = filterProps(firstPlan, ['name']);
```
Comment (arol) [inline controllers/users.controller.js]: `const`

File: controllers/plans.controller.js [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+    const user_id = ctx.user.id;
+    const plans = await this.Plan.findAll({
+      where: {
+        user_id
+      },
+      attributes: ['id', 'name']
+    });
+
+    if (plans) {
+      let res = [];
```
Comment (arol) [inline controllers/plans.controller.js]: `const`

**Review by arol — CHANGES_REQUESTED** [review]

**Comment by isadorabk** [comment]
push-ups done 💪💪💪

**Pull request merged**